lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -515,3 +525,30 @@ func getServiceAreaMaps(serviceAreaMaps map[string]lte_models.TacList) map[strin
 	}
 	return ret
 }
+
+// getFederatedNetworkConfigs in case this is a federated LTE networkm this function will try to parse out
+// feg_models.FederatedNetworkConfigs out of it
+func getFederatedNetworkConfigs(fegId lte_models.FegNetworkID, request *builder_protos.BuildRequest) (*feg_models.FederatedNetworkConfigs, error) {
+	if fegId == "" {
```

Comment:
I think it would also be good as a sanity check to check the request.Network.Type to ensure it is feg_lte.